lib/times_table.js

```diff
@@ -0,0 +1,15 @@
+function timesTable(numRows) {
+  let rows = '';
+
+  for (let i = 1; i <= numRows; i++) {
+    let row = '';
+    for (var j = 1; j <= numRows; j++) {
+      row += `${(i * j)}\t`;
+    }
+    rows += `${row}\n`;
```

Comment:
Is there an Array.prototype message you could research that could clean this up?

Reply:
Ya most likely. Let me look at it when I have time today.

Reply:
No problem! Just pushing you to think that way, at least for the future